napari/utils/event_handler.py

```diff
@@ -0,0 +1,13 @@
+from abc import abstractmethod
+
+
+class EventHandler:
```

Comment:
```suggestion
class EventHandler(ABC):
```
unlike the interface which had subclassing conflicts (right?) this one actually _can_ be a "real" abstract base class if we just just subclass ABC.  

EDIT:  but see my other comment wondering  why we need this _and_ LayerEventHandler

Reply:
To prepare us for having `ViewerEventHandler` and `DimsEventHandler`. Agreed, might be a bit thin right now though

Reply:
Ah, seeing point below, you're right - we could just make this a fully reusable `EventHandler`. Probably a good idea for now, we can easily make specific ones later

Reply:
dropping LayerEventHandler, so no longer needed